tests/scripts/kubeadm.sh

```diff
@@ -18,6 +18,14 @@ usage(){
 
 #install k8s master node
 install_master(){
+
+    # This is needed on K8S 1.6 to fix a regression. https://github.com/kubernetes/kubernetes/issues/47109
+    if [[ $KUBE_VERSION == v1.6* ]] ;
+    then
+        sudo sed -i 's/KUBELET_SYSTEM_PODS_ARGS=/KUBELET_SYSTEM_PODS_ARGS=--enable-controller-attach-detach=false /g' /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
```

Comment:
this sed expression seems like it would replace `KUBELET_SYSTEM_PODS_ARGS` multiple times even after its corrected. why not just add a new conf file `11-disable-attach-detach.conf`

Reply:
This is only called once. So it shouldnt be done multiple time. But ill add the new file and do it there.